contentcuration/contentcuration/frontend/channelEdit/components/ContentNodeListItem/index.spec.js

```diff
@@ -42,6 +42,11 @@ describe('ContentNodeListItem', () => {
       propsData: {
         node: DOCUMENT_NODE,
       },
+      computed: {
+        hasMetadataToDisplay() {
```

Comment:
This is looking to be an obsolete piece as there is no such computed in the `ContentNodeListItem`